judgments/api_client.py

```diff
@@ -86,16 +88,25 @@ def prepare_request_kwargs(
                 kwargs["params"] = data
             else:
                 kwargs["data"] = json.dumps(data)
+        if body is not None:
+            kwargs["data"] = body
         return kwargs
 
     def make_request(
-        self, method: str, path: str, get_multipart: False, data: Dict[str, Any] = None
+        self,
+        method: str,
+        path: str,
+        get_multipart: False,
+        data: Dict[str, Any] = None,
+        body=None,
     ) -> requests.Response:
-        kwargs = self.prepare_request_kwargs(method, path, data)
+        kwargs = self.prepare_request_kwargs(method, path, body, data)
         if get_multipart:
             headers = {"Accept": "multipart/mixed"}
         else:
             headers = {"Accept": "text/xml"}
+        if method == "PUT":
+            headers["Content-Type"] = "application/xml"
```

Comment:
@lozette If we find ourselves setting headers for each type of request, it might be better to tackle this in a different way. For example, maybe `make_request()` could accept a `headers` keyword argument, and the individual methods specify the extra headers they need?